src/main/java/wellnus/focus/feature/Countdown.java

```diff
@@ -87,10 +100,11 @@ public void start() {
         TimerTask countdownTask = new TimerTask() {
             @Override
             public void run() {
+                setIsReady(false);
                 if (!isRunClock.get()) {
                     return;
                 }
-                if (minutes == DEFAULT_STOP_TIME && seconds <= 10) {
+                if (minutes == DEFAULT_STOP_TIME && seconds <= 10 && seconds != 0) {
                     textUi.printOutputMessage(seconds + " seconds left");
                 }
```

Comment:
Can consider implying in the user guide that no commands are to be inputted when session is counting down. This might raise empty input or invalid input exceptions.

Reply:
Yes, this should be updated in the user guide.
Inputs are still possible though! In that case do we want to disallow input during this time? But I don't see any way to get around this...